src/main/java/org/humancellatlas/ingest/security/SecurityConfig.java

```diff
@@ -75,6 +75,7 @@ protected void configure(HttpSecurity http) throws Exception {
                 .antMatchers(HttpMethod.POST, "/submissionEnvelopes").authenticated()
                 .antMatchers(HttpMethod.POST, "/projects**").authenticated()
```

Comment:
At this point is it possible to determine which users are wranglers?
If so we can clear off: https://github.com/ebi-ait/hca-ebi-dev-team/issues/125

Reply:
The changes in this PR provides the basis for us to update the system to address #125. However, it does require modifications in the code, that's separate from the scope of the authorisation tasks.